src/app/components/publiclists/teamlist/teamlist.component.ts

```diff
@@ -449,31 +463,34 @@ export class TeamlistComponent implements OnInit {
 
   /**
    * Called after create team form is validated.
-   * @param self  context value of this.
-   */
-  createTeamSubmit(self) {
-    const API_PATH = self.createTeamsPath;
-    const url = self.globalService.formValueForLabel(self.components, 'team_url');
-    let TEAM_BODY: any = {
-      team_name: self.globalService.formValueForLabel(self.components, 'team_name')
-    };
-    if (url) {
-      TEAM_BODY['team_url'] = url;
+   * @param isvalidForm
+   */
+  createTeamSubmit(isvalidForm) {
+    if (isvalidForm) {
+      const API_PATH = this.createTeamsPath;
+      const url = this.create_team['team_url'];
+      let TEAM_BODY: any = {
+        team_name: this.create_team['team_name']
+      };
+      if (url) {
+        TEAM_BODY['team_url'] = url;
+      }
+      TEAM_BODY = JSON.stringify(TEAM_BODY);
+      this.globalService.startLoader('Creating Team');
+      this.apiService.postUrl(API_PATH, TEAM_BODY).subscribe(
+        data => {
+          this.globalService.stopLoader();
+          // Success Message in data.message
+          this.globalService.showToast('success', 'Team created successfully!', 5);
+          this.fetchMyTeams(this.fetchTeamsPath);
+        },
+        err => {
+          this.globalService.stopLoader();
+          this.globalService.handleFormError(this.components, err);
+        },
+        () => console.log('CREATE-TEAM-FINISHED')
```

Comment:
Please check for these statements in the whole PR.

Reply:
done